### ISSUE TITLE
Project wont build using CocoaPods 1.1.1

### ISSUE THREAD

**Issue by darlington-m** [Jaspersoft/js-ios-app#52]
Hi 
I cloned the project and tried to build it using the instruction on the Read me file. The initial issues i face when I run 'pod install' is:

```
Darlingtons-iMac:js-ios-app darlingtonmoyo$ pod install
Analyzing dependencies
[!] The dependency `Appirater (= 2.0.5)` is not used in any concrete target.
The dependency `SWRevealViewController (= 2.3.0)` is not used in any concrete target.
The dependency `JaspersoftSDK/JSCore (from `https://github.com/Jaspersoft/js-ios-sdk.git`, tag `2.5`)` is not used in any concrete target.
The dependency `JaspersoftSDK/JSSecurity (from `https://github.com/Jaspersoft/js-ios-sdk.git`, tag `2.5`)` is not used in any concrete target.
The dependency `JaspersoftSDK/JSReportExtention (from `https://github.com/Jaspersoft/js-ios-sdk.git`, tag `2.5`)` is not used in any concrete target.
Darlingtons-iMac:js-ios-app darlingtonmoyo$ 
```

I went past this by adding:

```
target 'TIBCO JasperMobile' do
 // appirater dependency
end 
```

After this change the dependencies are downloaded and i can open the project on Xcode fine. However, trying to run the project on the simulator induces this error:

```
d: library not found for -lPods 
clang: error: linker command failed with exit code 1 (use -v to see invocation) 
```

Im using Yosemite 10.10.5, git 2.3.8, CocoaPods 1.1.1

Grateful for any help
Thanks

**Comment by ogubariev** [comment]
Hi!
Could you talk which version of the app was downloaded?

**Comment by darlington-m** [comment]
I tried this using the 2.5 version source code and also I cloned the latest dev source code and the result was the same.

**Comment by ogubariev** [comment]
If you want to build the 2.5 version source code - you should use cocoapods v0.38.2. 
If you want to build the source code with the latest changes - you should checkout sources from 'other_improvements' branch and use cocoapods v1.1.1.

**Comment by darlington-m** [comment]
Im using cocoapods v1.1.1 and checked out the "other improvements" branch but i still get the same error. Here is the full stack trace:

```
Ld /Users/darlingtonmoyo/Library/Developer/Xcode/DerivedData/TIBCO_JasperMobile-hjzdpfrdcdmvsqfeymjxzazkeeip/Build/Products/Debug-iphonesimulator/TIBCO\ JasperMobile.app/TIBCO\ JasperMobile normal x86_64
    cd /Users/darlingtonmoyo/Desktop/js-ios-app
    export IPHONEOS_DEPLOYMENT_TARGET=8.0
    export PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator9.0.sdk -L/Users/darlingtonmoyo/Library/Developer/Xcode/DerivedData/TIBCO_JasperMobile-hjzdpfrdcdmvsqfeymjxzazkeeip/Build/Products/Debug-iphonesimulator -L/Users/darlingtonmoyo/Library/Developer/Xcode/DerivedData/TIBCO_JasperMobile-hjzdpfrdcdmvsqfeymjxzazkeeip/Build/Products/Debug-iphonesimulator/AFNetworking -L/Users/darlingtonmoyo/Library/Developer/Xcode/DerivedData/TIBCO_JasperMobile-hjzdpfrdcdmvsqfeymjxzazkeeip/Build/Products/Debug-iphonesimulator/Appirater -L/Users/darlingtonmoyo/Library/Developer/Xcode/DerivedData/TIBCO_JasperMobile-hjzdpfrdcdmvsqfeymjxzazkeeip/Build/Products/Debug-iphonesimulator/EasyMapping -L/Users/darlingtonmoyo/Library/Developer/Xcode/DerivedData/TIBCO_JasperMobile-hjzdpfrdcdmvsqfeymjxzazkeeip/Build/Products/Debug-iphonesimulator/JaspersoftSDK -L/Users/darlingtonmoyo/Library/Developer/Xcode/DerivedData/TIBCO_JasperMobile-hjzdpfrdcdmvsqfeymjxzazkeeip/Build/Products/Debug-iphonesimulator/SWRevealViewController -LJasperMobile/Vendor -LJasperMobile/Vendor/ALToastView-v0.0.1 -LJasperMobile/Vendor/GoogleAnalyticsServices -LJasperMobile/Vendor/PopoverView-v0.0.2 -LJasperMobile/Vendor/RNCachingURLProtocol -LJasperMobile/Vendor/VisualizeJS -LJasperMobile/Vendor/GoogleAnalyticsServices/GoogleAnalytics -LJasperMobile/Vendor/GoogleAnalyticsServices/GoogleAnalytics/Library -L/Users/darlingtonmoyo/Desktop/js-ios-app/JasperMobile/Vendor/GoogleAnalyticsServices -F/Users/darlingtonmoyo/Library/Developer/Xcode/DerivedData/TIBCO_JasperMobile-hjzdpfrdcdmvsqfeymjxzazkeeip/Build/Products/Debug-iphonesimulator -FJasperMobile/Vendor -FJasperMobile/Vendor/ALToastView-v0.0.1 -FJasperMobile/Vendor/GoogleAnalyticsServices -FJasperMobile/Vendor/PopoverView-v0.0.2 -FJasperMobile/Vendor/RNCachingURLProtocol -FJasperMobile/Vendor/VisualizeJS -FJasperMobile/Vendor/GoogleAnalyticsServices/GoogleAnalytics -FJasperMobile/Vendor/GoogleAnalyticsServices/GoogleAnalytics/Library -F/Users/darlingtonmoyo/Desktop/js-ios-app -filelist /Users/darlingtonmoyo/Library/Developer/Xcode/DerivedData/TIBCO_JasperMobile-hjzdpfrdcdmvsqfeymjxzazkeeip/Build/Intermediates/TIBCO\ JasperMobile.build/Debug-iphonesimulator/TIBCO\ JasperMobile.build/Objects-normal/x86_64/TIBCO\ JasperMobile.LinkFileList -Xlinker -rpath -Xlinker @executable_path/Frameworks -mios-simulator-version-min=8.0 -Xlinker -objc_abi_version -Xlinker 2 -ObjC -all_load -ObjC -lAFNetworking -lAppirater -lEasyMapping -lJaspersoftSDK -lSWRevealViewController -framework CFNetwork -framework CoreData -framework CoreGraphics -framework Foundation -framework MobileCoreServices -framework Security -framework SystemConfiguration -weak_framework StoreKit -fobjc-arc -fobjc-link-runtime -lz.1 -framework AssetsLibrary -lGoogleAnalyticsServices -lsqlite3.0 -framework Crashlytics -weak-lz -lxml2 -lPods -framework Fabric -framework WebKit -framework MessageUI -framework CFNetwork -framework CoreTelephony -framework QuartzCore -framework SystemConfiguration -framework MobileCoreServices -framework Security -framework CoreData -framework UIKit -framework Foundation -framework CoreGraphics -lPods-TIBCO\ JasperMobile -Xlinker -dependency_info -Xlinker /Users/darlingtonmoyo/Library/Developer/Xcode/DerivedData/TIBCO_JasperMobile-hjzdpfrdcdmvsqfeymjxzazkeeip/Build/Intermediates/TIBCO\ JasperMobile.build/Debug-iphonesimulator/TIBCO\ JasperMobile.build/Objects-normal/x86_64/TIBCO\ JasperMobile_dependency_info.dat -o /Users/darlingtonmoyo/Library/Developer/Xcode/DerivedData/TIBCO_JasperMobile-hjzdpfrdcdmvsqfeymjxzazkeeip/Build/Products/Debug-iphonesimulator/TIBCO\ JasperMobile.app/TIBCO\ JasperMobile

ld: library not found for -lPods
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by ogubariev** [comment]
Can you delete your copy of the app, fork it again and try to install pods?

**Comment by darlington-m** [comment]
Hi, I never had this issue resolved. I'm attempting to build again and still getting the same issue. Ive deleted and cloned several times, still no joy.

I have the same issue on an iMac Pro as well as MacBook Pro. 
On the MacBook I have  XCode 8.3.3 and Cocoapods 1.1.1
On the iMac I have XCode 7.0.1 Cocoapods 1.1.1

I cloned the repo and checked out "other_improvements" as recommended (also tried master and develop).

The first issue is that the Podfile doesn't seem to right. I don't know if you have tried installing the dependencies using the podfile recently. The issues I pointed out on the first comment are still the same one's I'm getting. 

Many thanks
Darlington
